docs/catalogue/cat_cohort-data-manager.md

```diff
@@ -1,15 +1,16 @@
-# Data manager of a cohort or data source
+# Data manager of a data resource
 
-## Data Catalogue
+## MOLGENIS catalogue
 
-[MOLGENIS Data Catalogue](https://data-catalogue.molgeniscloud.org/catalogue/catalogue/#) provides a
-framework to describe in detail: metadata of cohorts and of data sources; definitions of the variables collected in
-cohorts and data sources; and mappings to common data models. Its purpose is to facilitate pooled data analysis of multiple cohorts
-[Fortier et al, 2017](https://pubmed.ncbi.nlm.nih.gov/27272186/) and multi-data
-source studies [Gini et al, 2020](https://pubmed.ncbi.nlm.nih.gov/32243569/).
+[MOLGENIS catalogue](https://data-catalogue.molgeniscloud.org/catalogue/catalogue/#) provides a
+framework to describe in detail: metadata of different data resources, such as cohorts and data sources; definitions of 
+the variables collected in these resources; and mappings to common data models. Its purpose is to provide a gateway to 
+find and access health research data and to facilitate pooled data analysis of multiple cohorts 
+[Fortier et al, 2017](https://pubmed.ncbi.nlm.nih.gov/27272186/) and multi-data source studies 
+[Gini et al, 2020](https://pubmed.ncbi.nlm.nih.gov/32243569/).
 
-- The metadata of cohorts include descriptive information such as contact details, name of the cohort, and high-level
-  summary of contents and cohort design. The metadata of data sources, of the corresponding data banks and of the
+- The metadata of cohorts include descriptive information such as contact details, name of the resource, and a high-level
```

Comment:
'The metadata of cohorts' -> 'The metadata of resources'/'The metadata of data resources'